terraform/provider/provider.go

```diff
@@ -279,6 +316,43 @@ func (p *Provider) validateAddr(addr string, resp *tfsdk.ConfigureProviderRespon
 	return true
 }
 
+// getCredentialsFromBase64 returns client.Credentials built from base64 encoded keys
+func (p *Provider) getCredentialsFromBase64(certBase64, keyBase64, caBase64 string, resp *tfsdk.ConfigureProviderResponse) (client.Credentials, bool) {
+	cert, err := base64.StdEncoding.DecodeString(certBase64)
+	if err != nil {
+		resp.Diagnostics.AddError(
```

Comment:
Can we please add the env var name and config name?
I'm sure it will help with debugging